packages/cloud_firestore/cloud_firestore/example/lib/main.dart

```diff
@@ -114,7 +114,25 @@ class _FilmListState extends State<FilmList> {
 
     return Scaffold(
         appBar: AppBar(
-          title: Text('Firestore Example: Movies'),
+          title: Column(
+            mainAxisSize: MainAxisSize.min,
+            crossAxisAlignment: CrossAxisAlignment.stretch,
+            children: [
+              Text('Firestore Example: Movies'),
+
+              // This is a example use for "snapshots in sync".
+              // The view reflects the time of the last Firestore sync; which happens any time a field is updated.
+              StreamBuilder(
+                stream: FirebaseFirestore.instance.snapshotsInSync(),
```

Comment:
Not blocker but: It's not a good practice to create a new Stream directly inside the `build` method.

Since we're inside a StatefulWidget, this could be moved as property of `State` (and ideally same thing for the other streams in `build`

Reply:
Maybe `snapshotsInSync` should be a getter and cached instead, for convenience. But that's a breaking change

Reply:
I don't disagree, however I believe those example projects try to convene the point of how to use the API above best coding practices. The way it's written now only requires a developer to look at a single place to see how the streams are used. 

Moving it into `initState` will then require a `dispose()` call too

Reply:
We wouldn't need a dispose as there is nothing to dispose of.

It'd just be:

```dart
class MyState extends State<T> {
  final stream = FirebaseFirestore.instance.snapshotsInSync();
}
```


As an example, I believe it's important to show good practices, as other people will rely on those in their application. If the example uses a bad practice, that may make the bad practice propagate